sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/StateHolder.java

```diff
@@ -4,20 +4,30 @@
 
 import java.security.SecureRandom;
 import java.time.Duration;
+import java.util.Calendar;
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
+import java.util.Map.Entry;
 import java.util.concurrent.ConcurrentHashMap;
 
+import com.azure.spring.cloud.config.refresh.CalculatedBackoffTime;
+
 import com.azure.data.appconfiguration.models.ConfigurationSetting;
+import com.azure.spring.cloud.config.properties.AppConfigurationProviderProperties;
 
 final class StateHolder {
 
     private static final int MAX_JITTER = 15;
+
     private static final String FEATURE_ENDPOINT = "_feature";
+
     private static final Map<String, State> STATE = new ConcurrentHashMap<>();
+
     private static final Map<String, Boolean> LOAD_STATE = new ConcurrentHashMap<>();
 
+    private static Date nextForcedRefresh;
```

Comment:
Do you think we can use `Instant` here?

Reply:
I have updated in all locations. It makes it look like we have a breaking API change, but it isn't true as it is on an internal configuration file.